index/borrow.php

```diff
@@ -7,7 +7,8 @@
 <script src="/assets/js/bootstrap.min.js"></script>
 <script src="/assets/js/self-borrow.js"></script>
 <?php
-	require '../assets/api/checkuser.php';
+	require_once('../assets/api/dbconnector.php');
+	require_once '../assets/api/checkuser.php';
```

Comment:
On one hand, the code snippet from the php manual are all `User Contributed Notes` not official standards. AFAIK, pear coding standards is a more well known code styles for php. On the other hand, actually I don't mind if we set a rule that we must use parenthesis after `require_once`, the actual rule is we need to keep the SAME style.